### PR TITLE
Write gPLink abuse

### PR DESCRIPTION
**Pull request by JonasBK**
## Description

Original PR: https://github.com/SpecterOps/BloodHound/pull/567

This PR adds additional abuse info for Domain objects and the changes from running 'just generate'.

## Motivation and Context

See: BED-4430

This PR must be merged in and released in the same cycle as other PRs. See which in the BED-4430 ticket.

## How Has This Been Tested?

Tested locally with lab data.

## Types of changes

-   [ ] Chore (a change that does not modify the application functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [x] Documentation updates are needed, and have been made accordingly.
-   [ ] I have added and/or updated tests to cover my changes.
-   [x] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

**Comment by StephenHinck** [comment]
@q-roland - I just wanted to provide a quick update here for you. We're deep in the middle of some heavy SharpHound work that we must wrap up before merging in both PRs. I would expect that to wrap up here within the next few weeks. We really appreciate your contribution and patience - let me know if you have any questions!

File: packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericAll/WindowsAbuse.tsx [hunk 1]
```diff
@@ -423,13 +423,132 @@ const WindowsAbuse: FC<EdgeInfoProps & { targetId: string; haslaps: boolean }> =
...
+                    <Typography variant='body2'>
+                        If you want to be more targeted with your approach, it is possible to specify precisely what
+                        right you want to apply to precisely which kinds of descendent objects. You could, for example,
+                        grant a user "ForceChangePassword" permission against all user objects, or grant a security
+                        group the ability to read every GMSA password under a certain OU. Below is an example taken from
+                        PowerView's help text on how to grant the "ITADMIN" user the ability to read the LAPS password
+                        from all computer objects in the "Workstations" OU:
+                    </Typography>
+
+                    <Typography component={'pre'}>
```
Comment (urangel) [inline packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericAll/WindowsAbuse.tsx:489]: These lengthy code snippets might look nicer if you use the recently created `CodeController` component like in ADCSESC4 abuse sections.

**Review by urangel — APPROVED** [review]
Thank you, Jonas! 🙇

File: packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericAll/WindowsAbuse.tsx (hunk 1 above)
Comment (JonasBK) [reply packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericAll/WindowsAbuse.tsx:489]: Gotcha. Will check that out!

Comment (JonasBK) [reply packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericAll/WindowsAbuse.tsx:489]: Hmm.. It does look nicer with the code snippet style you have used ADCSESC4. I wonder if we should use for all code snippets everywhere to have consistent style? It is less relevant for one-liner commands though. Perhaps we should have a discussion with Ryan about it? If so, then we can create another ticket for that effort I guess.

**Comment by StephenHinck** [comment]
@q-roland, thank you again so much for your contributions! We're excited to incorporate this work into BloodHound with our next release!

I would love to send you a swag package to show our gratitude for your contributions to the product. If you can email me at shinck (AT) specterops (DOT) io with your shipping address, I'll get that on its way!

**Pull request merged**